bundle/bundle.go

```diff
@@ -374,29 +404,26 @@ func (r *Reader) Read() (Bundle, error) {
 
 	bundle.Data = map[string]interface{}{}
 
-	bundle.Signatures, descriptors, err = listSignaturesAndDescriptors(r.loader, r.skipVerify, r.sizeLimitBytes)
+	bundle.Signatures, bundle.Patch, descriptors, err = listSignaturesAndDescriptors(r.loader, r.skipVerify, r.sizeLimitBytes)
```

Comment:
nit: I'm a bit confused why we're reading the patch file contents inside `listSignaturesAndDescriptors` instead of below. I think this could be organized a bit differently. Why not treat the patch file as another descriptor?

Reply:
As discussed, renamed the function.